zkevm-circuits/src/copy_circuit/test.rs

```diff
@@ -19,6 +19,8 @@ use halo2_proofs::{
 };
 use mock::{test_ctx::helpers::account_0_code_account_1_no_code, TestContext, MOCK_ACCOUNTS};
 
+const K: u32 = 20;
+
```

Comment:
i was concern here that 20 may increase CI, but it now seems not too bad.  Acceptable.

Reply:
We were passing `k` values for each test, with a `max_copy_rows == 1000`. This just maintains the same `k` for each test, `max_copy_rows` default value was changed to `2000` (to allow pairing op copy bytes to be written).

If it's not much of a pain, we can stick with this?